fixtures/91-span-exec-nested.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "description": "otel-cli exec basic test and stuff",
+    "config": {
+      "env": {
+        "OTEL_EXPORTER_OTLP_ENDPOINT": "{{endpoint}}"
+      },
+      "cli_args": [
+        "exec",
+        "--service",
+        "main_test.go",
+        "--name",
+        "test-span-123",
+        "--kind",
+        "server",
+        "./otel-cli",
+        "exec",
+        "--tp-ignore-env",
+        "echo hello world $TRACEPARENT"
```

Comment:
As we discussed while pairing, we'll let this slide for now but the `sh -c` behavior might not remain the default. We can address it when/if this test breaks. Gotta make a firm decision on that soon, cuz if a bunch folks rely on it, it's gonna be unpleasant to take out.

Reply:
ack, it's a fine canary for when we decide